services/forecast_job_coordinator.py

```diff
@@ -7,42 +7,18 @@
 import pandas as pd
 from google.cloud import bigquery
 
-import utils
-from models import dataset, training_result
+from models import forecast_job_result, forecast_job_request
 from services import forecast_job_service
 
 
-@dataclasses.dataclass
-class ForecastJobRequest:
-    """An encapsulation of the training job request"""
-
-    # The unique key associated with a training method.
-    training_method: str
-
-    # The dataset used for model training.
-    dataset: dataset.Dataset
-
-    # The request start time.
-    start_time: datetime
-
-    # Parameters for training.
-    model_parameters: Dict[str, Any]
-
-    # Parameters for prediction.
-    prediction_parameters: Dict[str, Any]
-
-    # The unique request id.
-    id: str = dataclasses.field(default_factory=utils.generate_uuid)
-
-
 class ForecastJobCoordinator(abc.ABC):
     """
     Coordinates the queue of jobs, listing pending jobs and getting results.
     A forecast job is defined as a pipeline involved training a model, getting evaluations and getting a prediction.
     """
 
     @abc.abstractmethod
-    def enqueue_job(self, request: ForecastJobRequest) -> str:
+    def enqueue_job(self, request: forecast_job_request.ForecastJobRequest) -> str:
```

Comment:
Why not `from forecast_job_request import ForecastJobRequest`  and use only `ForecastJobRequest` in the type hint in this file?